### DISCUSSION TITLE
【求助】这卡强吗？

### DISCUSSION THREAD

**Discussion by xioagong** [share121/inter-knot#1684] [其他]
![274598537d30d13e0e5595d1d386bce](https://github.com/user-attachments/assets/62893524-f2f8-4fb1-8c54-7ac539ff57e5)
跟赛诺打娱乐，他先手掏了这个，纯霍普能打吗？

**Comment by mur-cat** [comment]
赶紧左上角快速胜利吧

**Comment by Philia093-T** [comment]
如果是你的，建议带个头盔玩；如果是对手玩，记得带把雾切去；但是如果对手是赛诺，建议光速投降，别牌也打不过人也打不过最后被气晕
![屑](https://github.com/user-attachments/assets/ec6edc0b-f68f-48f3-8b44-5ea75efe863f)
